src/Factory/ApiProblemStrategyFactory.php

```diff
@@ -4,9 +4,9 @@
 
 namespace Laminas\ApiTools\ApiProblem\Factory;
 
-use Interop\Container\ContainerInterface;
 use Laminas\ApiTools\ApiProblem\View\ApiProblemRenderer;
 use Laminas\ApiTools\ApiProblem\View\ApiProblemStrategy;
+use Psr\Container\ContainerInterface;
```

Comment:
This change is a potential breaking change for downstream extending classes: to avoid that, we probably need to include `laminas/laminas-servicemanager` as a polyfill :thinking:

Reply:
I added laminas/laminas-servicemanager to the composer.json file. Let me know if I should modify anything else.